styleguide/package.json

```diff
@@ -53,6 +53,6 @@
     "react-docgen-typescript-loader": "^3.1.0",
     "rimraf": "^3.0.0",
     "ts-jest": "^24.1.0",
-    "webpack": "^4.41.5"
+    "webpack": "4.41.2"
```

Comment:
Could we not bump the version of feeds-ui/package.json instead of downgrading all the others?

Reply:
See comment ^